### PR TITLE
[MODULAR] Resprites Latex Items + New additions

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
## About The Pull Request

Resprites various latex shoes, catsuits, and gloves. Adds the high heels to GAGs and adds a new pair of heels.

Atomizes this PR. #14522

## How This Contributes To The Skyrat Roleplay Experience

#14522

<details>
<summary>Ingame Sprites</summary>
<br>

image
image

Taur sprites exist, but right now taurs are currently broken anything taur related. So this bug will probally have to be fixed and this is an issue which remains outside of this PR.

https://github.com/Skyrat-SS13/Skyrat-tg/issues/14605

</details>

## Changelog

:cl:
add: High Heels are now added to GAGs
add: Black heels are added into the loadout
add: Latex bra, panties, and socks are added to the underwear menu.
balance: Corset and Chaps are no longer ludicriously expensive
fix: Colourable cloth wraps now show color on digigrade species.
imageadd: New Various Latex Sprites
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 1]
```diff
@@ -378,6 +378,22 @@
 /datum/greyscale_config/wraps/worn
 	name = "Cloth Wraps (Worn)"
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/wraps/wraps_worn.json'
+/datum/greyscale_config/wraps/worn/digi
```
Comment (Zonespace27) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm:382]:
```suggestion

/datum/greyscale_config/wraps/worn/digi
```

**Comment by Jolly-66** [comment]
If you added/updated any new clothing sprites, it's common practice to include them in your PR so we may see what they look like on players 

I would recommend putting them in a spoiler so it doesn't make your PR body long (of course, assuming there isn't a lot of changes, and you can always do multiple in one screenshot)

**Comment by StrangeWeirdKitten** [comment]
> If you added/updated any new clothing sprites, it's common practice to include them in your PR so we may see what they look like on players
> 
> I would recommend putting them in a spoiler so it doesn't make your PR body long (of course, assuming there isn't a lot of changes, and you can always do multiple in one screenshot)

I had all the screen shots listed in the linked PR. Do you want actual in game screen shots of the various items?

**Comment by Jolly-66** [comment]
> Strange said something 

Ah, I didn't see that PR originally, apologies.

I'm no spritetainer nor is it my realm to exercise my hand, but it would be beneficial to have in-game screenshots.

Please feel free to do whatever with what I said. Ultimately, it's up to you and not me here, but would be appreciated no less!

**Comment by StrangeWeirdKitten** [comment]
Taur sprites are broken outside of this PR. So I made a bug report. But the sprites overall will exist if style_taur_all works again.

File: modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm [hunk 2]
```diff
@@ -39,6 +39,19 @@
...
+	greyscale_colors = "#FFFFFF"
+	flags_1 = IS_PLAYER_COLORABLE_1
+	body_parts_covered = TRUE
+
+/obj/item/clothing/shoes/black_heels
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
+	name = "black heels"
+	desc = "A pair of black high heels that are much smaller on your feet"
+	icon_state = "highheels"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm]: We typically always put `name` and `desc` first, and try to basically put the long variables lower than the short ones, when possible, to make easier to read through.
```suggestion
	name = "black heels"
	desc = "A pair of black high heels that are much smaller on your feet"
	icon_state = "highheels"
	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
```

File: modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm [hunk 3]
```diff
@@ -39,6 +39,19 @@
 	name = "high heels"
 	desc = "A fancy pair of high heels. Won't compensate for your below average height that much."
 	icon_state = "heels"
+	greyscale_config = /datum/greyscale_config/heels
+	greyscale_config_worn = /datum/greyscale_config/heels/worn
+	greyscale_config_worn_digi = /datum/greyscale_config/heels/worn/digi
+	greyscale_colors = "#FFFFFF"
+	flags_1 = IS_PLAYER_COLORABLE_1
+	body_parts_covered = TRUE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm]: Are you sure this is a TRUE/FALSE kind of variable, and not a flag-based variable (which I suspect it is)?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Pull request merged**